Tests/SonarScanner.Shim.Tests/PropertiesWriterTest.cs

```diff
@@ -53,6 +53,7 @@ public void PropertiesWriterToString()
             string productFile = CreateEmptyFile(productBaseDir, "File.cs");
             string productChineseFile = CreateEmptyFile(productBaseDir, "你好.cs");
 
+            string productFxCopFilePath = CreateEmptyFile(productBaseDir, "productFxCopReport.txt");
```

Comment:
Is it still needed? We dropped support for fxcop.

Reply:
Probably I missed that... I copy-pasted my files on top of JH's changes and then manually reverted changes.